nemo/collections/nlp/models/language_modeling/megatron/bert_model.py

```diff
@@ -71,14 +72,23 @@ class BertLMHead(MegatronModule):
     """
 
     def __init__(
-        self, mpu_vocab_size, hidden_size, init_method, layernorm_epsilon, parallel_output, use_openai_gelu, onnx_safe
+        self,
+        mpu_vocab_size,
+        hidden_size,
+        init_method,
+        layernorm_epsilon,
+        parallel_output,
+        use_openai_gelu,
+        onnx_safe,
+        sequence_parallel,
```

Comment:
Can you set default values for some of these parameters?

Reply:
Yes, I am setting on sequence parallel alone, and leaving others just as was before.